examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>This warns</p><p> </p><p>08:53:51|PASEOS-WARNING| <strong>You provided a different starting epoch for PASEOS than the local time of the local actor.starting_epoch will be used.</strong></p><p> </p><p>why?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>In the cell before</p><p> </p><pre class="ql-syntax" spellcheck="false">Line #5.    sat = ActorBuilder.get_actor_scaffold(name="sat", actor_type=SpacecraftActor, epoch=pk.epoch(0))
</pre><p>Here you define the start time as pk.epoch(0) not start date ;)</p>

Reply:
In the cell above

Line #5.    sat = ActorBuilder.get_actor_scaffold(name="sat", actor_type=SpacecraftActor, epoch=pk.epoch(0))

Here you define the start time as pk.epoch(0) not start date ;)